metrics/src/main/java/com/megaease/easeagent/metrics/MetricRegistryService.java

```diff
@@ -32,7 +36,27 @@
     public MetricRegistry createMetricRegistry() {
         MetricRegistry registry = new MetricRegistry();
         REGISTRY_LIST.add(registry);
-        new DropwizardExports(registry).register();
+        new DropwizardExports(registry, MetricFilter.ALL, new EaseAgentSampleBuilder()).register();
         return registry;
     }
+
+    class EaseAgentSampleBuilder extends DefaultSampleBuilder {
+
+        @Override
+        public Collector.MetricFamilySamples.Sample createSample(String dropwizardName, String nameSuffix, List<String> additionalLabelNames, List<String> additionalLabelValues, double value) {
+            return super.createSample(rebuildName(dropwizardName), nameSuffix, additionalLabelNames, additionalLabelValues, value);
```

Comment:
What are the values of the additionalLabelValues? We expect all metrics with labels which are service_name, host_ipv4, system, etc...

Reply:
By default, their label is null, which is only given by the server of prometheus. Of course, we can also put our general label on it according to the actual situation. I'll add it now